api/db/seeds/data/rihanna-certified-profile-builder.js

```diff
@@ -0,0 +1,1299 @@
+  databaseBuilder.factory.buildKnowledgeElement({ source: "inferred", skillId: "recsOcGsdjIW4IpiV", earnerPix: 0.8, status: "validated", createdAt: new Date("2020-01-30 14:02:18.1+00"), competenceId: "recudHE5Omrr10qrx", answerId, assessmentId, userId });
+  answerId = databaseBuilder.factory.buildAnswer({ result: "ok", value: "Anne-Gaëlle vient de sniper Laura avec un garde !", challengeId: "recsZFlvRHurkUsqF", assessmentId, createdAt: new Date("2020-01-30 14:02:18.06+00"), updatedAt: new Date("2020-01-30 14:02:18.06+00"), elapsedTime: 0, resultDetails: "nume1: true\nnume4: true\nnume2: true\nnume5: true\n" }).id;
+  databaseBuilder.factory.buildKnowledgeElement({ source: "inferred", skillId: "recinOcg9j0QI6MJS", earnerPix: 2, status: "validated", createdAt: new Date("2020-01-30 14:02:18.1+00"), competenceId: "recudHE5Omrr10qrx", answerId, assessmentId, userId });
+  answerId = databaseBuilder.factory.buildAnswer({ result: "ok", value: "Anne-Gaëlle vient de sniper Laura avec un garde !", challengeId: "recsZFlvRHurkUsqF", assessmentId, createdAt: new Date("2020-01-30 14:02:18.06+00"), updatedAt: new Date("2020-01-30 14:02:18.06+00"), elapsedTime: 0, resultDetails: "nume1: true\nnume4: true\nnume2: true\nnume5: true\n" }).id;
+  databaseBuilder.factory.buildKnowledgeElement({ source: "inferred", skillId: "recbhCa10f0cXOdbD", earnerPix: 1, status: "validated", createdAt: new Date("2020-01-30 14:02:18.1+00"), competenceId: "recudHE5Omrr10qrx", answerId, assessmentId, userId });
+  answerId = databaseBuilder.factory.buildAnswer({ result: "ok", value: "Anne-Gaëlle vient de sniper Laura avec un garde !", challengeId: "recsZFlvRHurkUsqF", assessmentId, createdAt: new Date("2020-01-30 14:02:18.06+00"), updatedAt: new Date("2020-01-30 14:02:18.06+00"), elapsedTime: 0, resultDetails: "nume1: true\nnume4: true\nnume2: true\nnume5: true\n" }).id;
+  databaseBuilder.factory.buildKnowledgeElement({ source: "direct", skillId: "recCBkEOJVHpKos5m", earnerPix: 0.8, status: "validated", createdAt: new Date("2020-01-30 14:02:18.099+00"), competenceId: "recudHE5Omrr10qrx", answerId, assessmentId, userId });
+  answerId = databaseBuilder.factory.buildAnswer({ result: "ok", value: "Anne-Gaëlle vient de sniper Laura avec un garde !", challengeId: "recQLrkjETyhe4HJN", assessmentId, createdAt: new Date("2020-01-30 14:02:17.032+00"), updatedAt: new Date("2020-01-30 14:02:17.032+00"), elapsedTime: 0, resultDetails: "null\n" }).id;
+  databaseBuilder.factory.buildKnowledgeElement({ source: "inferred", skillId: "recxtb5aLs6OAAKIg", earnerPix: 1.14286, status: "validated", createdAt: new Date("2020-01-30 14:02:17.591+00"), competenceId: "re
```

Comment:
Comment a été généré les KE par rapport aux answers ? 
Je vois dans le code des Answers qui créent des KE inférré sans KE direct. 
Ma crainte est que ces possibles raccourci fassent que le profil créé n'est pas totalement stable pour des tests.

Reply:
@MelanieMEB D'abord, my bad, c'est encore en development in progress. Ensuite, les données que tu vois sont issues de "vraies" données. En gros, sur intégration, j'ai demandé à anne-so à l'aide des boutons magiques de créer un profil niveau 5 partout. Puis j'ai pompé TOUS ses KE + answers. Normalement y'a rien d'incohérent, c'est de la copie de data pure et dure.

Reply:
:+1: Pas de problème, ça a l'air bon comme ça :) Merci pour ces seeds !